lib/platform/bitbucket-server/index.js

```diff
@@ -428,7 +428,7 @@ async function getPrList() {
     const values = await utils.accumulateValues(
       `/rest/api/1.0/projects/${config.projectKey}/repos/${
         config.repositorySlug
-      }/pull-requests?state=OPEN`
+      }/pull-requests?state=ALL&limit=100`
```

Comment:
Will this still paginate to get all?

Reply:
Yes. I've widened the limit from 25 to 100 like github.